integration/examples/jib-sync/README.md

```diff
@@ -0,0 +1,165 @@
+### Example: Jib Sync
+
+[Jib](https://github.com/GoogleContainerTools/jib) is one of the supported builders in Skaffold. Jib
+has special sync support using the `auto` configuration.
+
+## Running the example
+
+Run the maven or gradle version of the example with port forwarding on.
+
+#### Gradle
+```
+$ skaffold dev -f skaffold-gradle.yaml --port-forward
+```
+
+#### Maven
+```
+$ skaffold dev -f skaffold-maven.yaml --port-forward
+```
+
+You can now see sync in action:
+1. See the original response from the `HelloController` in the spring application 
+  ```
+  $ curl localhost:8080
+  text-to-replace
+  ```
+1. Edit the hello controller `src/main/java/hello/HelloController.java`
+  ```diff
+  +       return "some-new-text";
+  -       return "text-to-replace";
+  ```
+1. Give skaffold a few seconds to synchronize the file to the container, and give Spring
+   Boot Developer Tools a chance to reload your application.
+1. See the updated response from the `HelloController`
+  ```
+  $ curl localhost:8080
+  some-new-text
+  ```
+1. You've now seen auto sync in action!
+
+## How it works
+
+This example contains both maven and gradle build configs and separate skaffold.yamls.
+
+*gradle*: `skaffold-gradle.yaml`
+*maven*: `skaffold-maven.yaml`
+
+use the `-f` flag to specify the correct buildfile when running (or rename your preferred option to `skaffold.yaml`)
+```
+$ skaffold -f skaffold-gradle.yaml ...
+```
+
+We configure it in `skaffold.yaml`, by enabling sync on the jib artifact.
+
+```yaml
+build:
+  artifacts:
+  - image: skaffold-example
+    context: .
+    jib: {}
+    sync: 
+      auto: {}
+```
+
+This example is designed around the functionality available in [Spring Boot Developer Tools](https://docs.spring.io/spring-boot/docs/current/reference/html/using-spring-boot.html#using-boot-devtools) for developing against running applications.
+
+Some additional steps in your java build are required for this to work:
+- Sync requires `tar` on the running container to copy files over. The default base image that Jib uses `gcr.io/distroless/java` does not include `tar` or any utilities. During development you must use a base image that includes `tar`, in this example we use the `debug` flavor of distroless: `gcr.io/distroless/java:debug` 
+`maven`
```

Comment:
`maven` is not properly rendered. Probably put a new line on it.

Reply:
This is still not correctly rendered. `maven` is on the same line with `gcr.io/distroless/java:debug`.

Screenshot:

![image](https://user-images.githubusercontent.com/10523105/85319934-4c807a00-b490-11ea-9c37-07efde66ac6a.png)